formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -336,4 +336,4 @@ type classAttributesOnKeys = <
   key2 [@bs.get null] : (Js.t int) [@onType2],
   key3 [@bs.get null] : (Js.t int) [@onType2],
   key4 : Js.t (int [@justOnInt])
->;
+ >;
```

Comment:
This causes strange indenting though! But don't worry, @SanderSpies diff for new syntax for object types makes this obsolete anyways (and we can delete @yunxing's token splitting hack in the lexer/parser).